src/pages/dar_application/ResearcherInfo.js

```diff
@@ -84,14 +85,15 @@ export default function ResearcherInfo(props) {
 
         div({className: 'dar-application-row'}, [
           h3('1.2 Researcher Identification' + (formData.checkCollaborator ? ' (optional)' : '')),
-          span({ className: `${showNihValidationError ? 'errored' : 'default-color'}`, isRendered: formData.checkCollaborator !== true }, [
+          !readOnlyMode && span({ className: `${showNihValidationError ? 'errored' : 'default-color'}`, isRendered: formData.checkCollaborator !== true }, [
```

Comment:
I'm still seeing this alert, but I think it's coming from line around line 109. The alert code is essentially a copy of the `missingLibraryCard` constant on line 16.

Reply:
<img width="916" alt="Screenshot 2023-10-19 at 10 10 12 AM" src="https://github.com/DataBiosphere/duos-ui/assets/116679/6af3c418-2354-468f-a79a-70ea2e90f650">

Reply:
Sorry, it should be fixed now!